src/clj/game/core/cost_fns.clj

```diff
@@ -44,8 +45,13 @@
   ([state side card] (rez-additional-cost-bonus state side card nil))
   ([state side card pred]
    (let [costs (merge-costs
-                 [(:additional-cost card)
-                  (:additional-cost (card-def card))
+                 ;; idk what the difference is between the additional cost for the card,
+                 ;; and the additional cost for the cdef, but we should probably actually
+                 ;; have a note here explaining it (somewhere in this file)
```

Comment:
There's no difference, just two different places they're tracked. I don't remember why/how `:additional-cost` can be attached to a card. A quick search makes me think we don't do that, so you could try removing it and seeing what breaks.

Reply:
ok this one does actually break the `get-rez-cost-test` (`rezzing_test.clj:24,26`), both of them mentioning `:trash-can` as one of the costs.
My base thought is this has something to do with abilities that have a fake trash can cost or something of the like, idk.

Reply:
yeah, that's just a test checking the functionality works. that part of the test can be deleted and this removed.